coral/src/services/api-mocks/server.ts

```diff
@@ -0,0 +1,9 @@
+import { setupServer } from "msw/node";
+import { rest } from "msw";
+
+export const server = setupServer(
+  rest.all("*", (req, res, ctx) => {
+    console.error("Please mock all api calls in your tests.");
```

Comment:
I would rather throw an aggressive error and fail the tests. I think we should require all API calls to be mocked in order to run the tests successfully. Other option would be to add [jest-fail-on-console](https://github.com/ValentinH/jest-fail-on-console) or similar plugin, but lets leave that to later. 

```
throw Error(`Unhandled API request: ${req.method} ${req.url}`)
```

Reply:
Unfortunately throwing an Error does not work probably in this setup. It leads to an `Network request failed` error in `whatwg-fetch`, which is not really giving a lot of information in the test. I tried it in all variants and stages I could think of. 

In general I'm 1000000% on breaking things really hard for that kind of errors in tests 😬 , I just didn't get it to work in a way that provides helpful feedback to a dev.

Reply:
Alright. So probably a console.error with that jest plugin is the solution.